### PR TITLE
ci: Don't build kubernetes olm bundle

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatolii Bazko <abazko@redhat.com>

### What does this PR do?
Don't build kubernetes olm bundle anymore.

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/20668

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [Custom resource definition file is up to date](https://github.com/eclipse-che/che-operator/blob/main/README.md#updating-custom-resource-definition-file)
- [x] [OLM bundles are up to date](https://github.com/eclipse-che/che-operator/blob/main/README.md#update-olm-bundle)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by mmorhun — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -467,12 +467,12 @@ update-roles:
 .PHONY: bundle
 bundle: generate manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
 	if [ -z "$(platform)" ]; then
-		echo "[ERROR] You must specify 'platform' macros. For example: `make bundle platform=kubernetes`"
+		echo "[ERROR] You must specify 'platform' macros. For example: `make bundle platform=openshift`"
```
Comment (AndrienkoAleksandr) [inline Makefile:470]: Sorry, why do we need platform argument if we will support only on platform Openshfit ?

File: Makefile [hunk 2]
```diff
@@ -780,14 +721,10 @@ get-next-version-increment:
 
 update-resources: SHELL := /bin/bash
 update-resources: check-requirements update-resource-images update-roles update-helmcharts
-	for platform in 'openshift' 'kubernetes'
+	for platform in 'openshift'
```
Comment (AndrienkoAleksandr) [inline Makefile:724]: I think we can remove this cycle for if we're supporting only one platform.

File: olm/README.md [hunk 3]
```diff
@@ -13,29 +13,26 @@ WARNING: Please make sure to use the precise `v1.7.1` version of the `operator-s
 
 ## Eclipse Che OLM bundles
 
-OLM (operator lifecycle manager) provides ways of installing operators. One of the convenient way how to achieve it is by using OLM bundles. See more about the format: https://github.com/openshift/enhancements/blob/master/enhancements/olm/operator-bundle.md. There two "next" platform-specific OLM bundles for Сhe operator:
+OLM (operator lifecycle manager) provides ways of installing operators. One of the convenient way how to achieve it is by using OLM bundles. See more about the format: https://github.com/openshift/enhancements/blob/master/enhancements/olm/operator-bundle.md. There two OLM bundles:
```
Comment (AndrienkoAleksandr) [inline olm/README.md:16]: I think we also can metion about next-all-namespace bundle.

Comment (tolusha) [reply olm/README.md:16]: tech-preview channel will removed soon

File: Makefile (hunk 2 above)
Comment (tolusha) [reply Makefile]: done

File: Makefile (hunk 1 above)
Comment (tolusha) [reply Makefile]: fixed

**Pull request merged**